test/unit/app/renderer/components/download/downloadBarTest.js

```diff
@@ -92,11 +126,11 @@ describe('downloadsBar component', function () {
     })
   })
 
-  describe('very narrow downloads bar with items', function () {
+  // skipped until #9176 is merged
+  describe.skip('very narrow downloads bar with items', function () {
```

Comment:
@luixxiul no need, because I will include it in #9176, that will be part of 0.18 as well